src/routes/blog/post/case-study-majik-kids/+page.markdoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Majik Kids: Building an audio-based content platform for children"
+description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, illustrators, etc., to build an alternative content platform with a focus on audio storytelling, creating a healthier learning experience for children.
+date: 2024-01-19
+cover: /images/blog/majik-kids.png
+timeToRead: 5
```

Comment:
This is not a 5 minute read haha

Reply:
A fit it into an online time-to-read calculator and it gave <6 xD

Updating it to 7